src/Mvc/test/ViewCompilationApps/Directory.Build.targets

```
@@ -1,23 +1,23 @@
 <Project>
-  <Import Project="..\Directory.Build.targets" />
+  <Import Project="..\..\..\..\Directory.Build.targets" />
```

Automatically chain the parent whenever possible. IF there is an explicit reason parents should be skipped, it's better to comment why or override the parent.
```suggestion
<Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.targets))\Directory.Build.targets" />
```

---

Even if that file doesn't exist?

---

There will always be, at the very least, a Directory.Build.targets file in the repo root. GetDirectoryNameOfFileAbove searches upwards for the nearest parent.